.gitignore

```diff
@@ -9,3 +9,4 @@
 # based on the name of the directory bazel is cloned into.
 /bazel-*
 .DS_Store
+.DS_Store
```

Comment:
The `.DS_Store` file has already been checked in. You can remove this from gitignore, instead delete the .DS_Store file and git add so that git marks the file as removed.